common/postgresql/values.yaml

```diff
@@ -104,18 +104,6 @@ upgrades:
   revisionHistory: 3
   podReplacementStrategy: Recreate
 
```

Comment:
```suggestion
## Configure resource requests and limits	
## ref: http://kubernetes.io/docs/user-guide/compute-resources/	
##	
resources: {}
#  limits:	
#    memory: 256Mi	
#    cpu: 500m	
#  requests:	
#    memory: 256Mi	
#    cpu: 500m	
```

Reply:
Defaults to empty resources. Guards against rendering error in the `toYAML` function. Gives an example of the expected values